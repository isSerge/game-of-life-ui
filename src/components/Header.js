import React from 'react'
import styled from 'styled-components'

const Wrapper = styled.header`
    display: flex;
    justify-content: space-between;
    margin: 20px 0;
`

const Title = styled.h1`
    margin: 0;
    text-transform: uppercase;
`

const Header = ({ title }) => (
    <Wrapper>
        <Title>{title}</Title>
    </Wrapper>
)

export default Header
