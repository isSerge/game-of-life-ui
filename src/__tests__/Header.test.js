import React from 'react'
import Header from '../components/Header'
import renderer from 'react-test-renderer'

test('Header matches snapshot', () => {
    const title = 'Random title'
    const component = renderer.create(<Header title={title} />)
    let tree = component.toJSON()

    expect(tree).toMatchSnapshot()
})
