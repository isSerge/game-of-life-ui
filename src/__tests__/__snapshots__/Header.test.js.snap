// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header matches snapshot 1`] = `
<header
  className="sc-bdVaJa fbVJlx"
>
  <h1
    className="sc-bwzfXH jEoixz"
  >
    Random title
  </h1>
</header>
`;
